jina/executors/indexers/keyvalue.py

```diff
@@ -234,6 +240,26 @@ def _delete(self, keys: Iterable[str]) -> None:
                 del self.query_handler.header[key]
             self._size -= 1
 
+    def _rollback_delete(self, keys, values):
+        self.query_handler.close()
```

Comment:
Let's see if we can wait for https://github.com/jina-ai/jina/pull/2106 to be merged, so that we can use `handlers` properly

Reply:
#2106 is not going to happen any time soon, I think. It's trickier than we thought and some other stuff got prioritized